core-bundle/contao/classes/Versions.php

```diff
@@ -671,6 +671,7 @@ public static function addToTemplate(BackendTemplate $objTemplate)
 								   ->execute(...$params);
 
 		$security = System::getContainer()->get('security.helper');
+		$request = System::getContainer()->get('request_stack')->getMainRequest();
```

Comment:
Should this be `getCurrentRequest()` based on #5006?

Reply:
Agree.
```suggestion
		$request = System::getContainer()->get('request_stack')->getCurrentRequest();
```